app/models/manageiq/providers/openstack/storage_manager/swift_manager.rb

```diff
@@ -27,6 +27,10 @@ class ManageIQ::Providers::Openstack::StorageManager::SwiftManager < ManageIQ::P
     end
   end
 
+  class << self
+    delegate :refresh_ems, :to => ManageIQ::Providers::Openstack::CloudManager
+  end
+
```

Comment:
It looks like Swift isn't refreshed with (cloud+network+cinder) so we should leave this one out